paddle/fluid/pybind/inference_api.cc

```diff
@@ -55,6 +57,7 @@ void BindInferenceApi(py::module *m) {
   BindNativePredictor(m);
   BindAnalysisConfig(m);
   BindAnalysisPredictor(m);
+  BindMkldnnQuantizerConfig(m);
```

Comment:
Please, wrap this line with `#ifdef PADDLE_WITH_MKLDNN`

Reply:
Done